includes/course-theme/class-sensei-course-theme.php

```diff
@@ -466,12 +466,14 @@ public function add_lesson_video_to_content( $content ) {
 			return $content;
 		}
 
-		remove_filter( 'the_content', [ $this, 'add_lesson_video_to_content' ], 80 );
-
 		ob_start();
 		Sensei()->frontend->sensei_lesson_video( get_the_ID() );
 		$video = ob_get_clean();
 
-		return $video . $content;
+		if ( ! empty( $video ) && false === strpos( $content, $video ) ) {
```

Comment:
I would suggest adding a comment or something documenting why this code is required (e.g in this case, it could be something like "fixes a compatibility issue with Yoast SEO where the video wouldn't appear in learning mode", maybe linking the issue or the PR).

While we have git to know the story behind each line, I **think** this might be a good practice especially for those cases where the incompatibility by itself is kinda..unexpected. 

WDYT?

Reply:
WDYT about the comment I added here: https://github.com/Automattic/sensei/pull/5044/commits/c29b5b7556408e7a189b0d1d3f94c736bfa60316?

It's a more generic comment (not related to Yoast) because it's not related specifically to that plugin. Just a different approach that also works with Yoast.

Reply:
It looks good to me! Thanks! :)